skyvern/config.py

```diff
@@ -61,14 +61,15 @@ class Settings(BaseSettings):
     # LLM Configuration #
     #####################
     # ACTIVE LLM PROVIDER
-    LLM_KEY: str = "OPENAI_GPT4V"
```

Comment:
Can we keep the default llm_key as `OPENAI_GPT4V` as this is the most reliable model that's been well tested by the skyvern team.